src/classes/AuthenticationSession.ts

```diff
@@ -15,10 +15,16 @@ enum FusionAuthStatusCode {
 export class AuthenticationSession {
   public authToken = '';
 
+  public refreshToken = '';
+
   public readonly authContext;
 
+  private authTokenExpiresAt = 0;
+
   private readonly fusionAuthClient;
 
+  private readonly fusionAuthAppId = process.env.FUSION_AUTH_APP_ID ?? '';
```

Comment:
This should either be a static attribute of an appropriate class (this class is fine) *OR* it should be passed as part of the constructor (e.g. `sftpFusionAuthAppId`) <-- also I'd specify in the name of the variable AND the env variable that it is intended to be the **sftp** fusion auth app ID (since there is more than one application id a dev could pick from)